Makefile

```diff
@@ -17,7 +19,12 @@ build:
 image:
 	imagebuilder -t openshift/telemeter:latest .
 
-check:
+lint: $(GOLANGCI_LINT_BIN)
+	# megacheck fails to respect build flags, causing compilation failure during linting.
+	# instead, use the unused, gosimple, and staticcheck linters directly
+	$(BIN)/golangci-lint run -e '.*Write' -D megacheck -E unused,gosimple,staticcheck
```

Comment:
honestly I am not sure about ignoring `.*Write` method signatures not checking errors. We can enable this, the code just becomes a bit chatty in some places with `_ = Write...`.

Reply:
I don’t much like the idea of ignoring the errors in `.Write...`. We can ignore the byte counts but even if we don’t return the error, at least log it. Go2’s error handling will make this feel less “chatty” 😉 the insight into write failures can be very informative about network, memory, or disk issues.